shell/platform/windows/keyboard_key_embedder_handler.cc

```diff
@@ -264,8 +264,14 @@ void KeyboardKeyEmbedderHandler::KeyboardHookImpl(
     pressingRecords_[physical_key] = eventual_logical_record;
   } else {
     auto record_iter = pressingRecords_.find(physical_key);
-    assert(record_iter != pressingRecords_.end());
-    pressingRecords_.erase(record_iter);
+    // Critical keys can be unset before this poitn if get_key_state_ finds they are not
+    // currently pressed. If a the physical key of a keyup event is not in pressingRecords_,
+    // only assert that it is a critical key.
```

Comment:
Maybe clarify that an example of this scenario is when the caps lock key is pressed twice quickly while Windows Narrator is enabled.